extensions/amp-fit-text/0.1/amp-fit-text.js

```diff
@@ -236,6 +219,47 @@ export function updateOverflow_(content, measurer, maxHeight, fontSize) {
   });
 }
 
+/**
+ *
+ * @param {!Document} document
+ * @param {!Element} element
+ */
+export function buildDom(document, element) {
+  const content = document.createElement('div');
+  applyFillContent(content);
+  content.classList.add(CONTENT_CLASS);
+
+  const contentWrapper = document.createElement('div');
+  contentWrapper.classList.add(CONTENT_WRAPPER_CLASS);
+  content.appendChild(contentWrapper);
+
+  const measurer = document.createElement('div');
+  measurer.classList.add(MEASURER_CLASS);
+
+  realChildNodes(element).forEach((node) => contentWrapper.appendChild(node));
+  copyInnerHtml(contentWrapper, measurer);
+  element.appendChild(content);
+  element.appendChild(measurer);
+}
+
+/**
+ * @param {!Element} element
+ * @param {string} className
+ * @return {Element}
+ */
+function getDescendentByClass(element, className) {
+  return scopedQuerySelector(element, ` .${escapeCssSelectorIdent(className)}`);
```

Comment:
Über-nit
```suggestion
  return scopedQuerySelector(element, `.${escapeCssSelectorIdent(className)}`);
```

Reply:
Isn't the whitespace intentional to scope to the element descendants?

Reply:
The space is actually required for it to become a descendent selector

Reply:
I don't believe that to be true. This translates to `element.querySelector(prependSelectorsWith('.className', ':scope'))`. `prependSelectorsWith(selector, distribute)` puts a space between `distribute` and `selector`, as the code comment examples show (https://github.com/ampproject/amphtml/blob/main/src/core/dom/css-selectors.js#L83).

Reply:
Result when I remove the space:
```
extensions/amp-fit-text/0.1/amp-fit-text.js
  251:10  error  using scopedQuerySelector here is actually unnecessary, since you don't use child selector semantics  local/query-selector
```

Reply:
I think that's telling you that since you're not using child (`> foo`) or descendent (`div div`), you can just use `element.querySelectorAll(`.${...}`);` here.

See explanation for that lint rule here: https://github.com/ampproject/amphtml/blob/main/build-system/eslint-rules/query-selector.js#L77

Reply:
Thanks for continuing to push here! Switched to `element.querySelector`